core/src/lib.rs

```diff
@@ -7,7 +7,7 @@ extern crate alloc;
 use core::ops::Range;
 
 pub mod chiplets;
-pub use chiplets::{bitwise, hasher};
+pub use chiplets::hasher;
```

Comment:
I'm assuming removing this is left for future PRs, right?

Reply:
Yes, exactly. I figured I'd do it with the hasher updates.